components/camel-sip/src/main/java/org/apache/camel/component/sip/SipConfiguration.java

```diff
@@ -408,19 +731,15 @@ Properties createInitialProperties() {
         return properties;
     }
 
-    public AddressFactory getAddressFactory() {
```

Comment:
@nikvaessen many of these getter methods have been re-ordered, this makes the diff much longer than necessary, could you make another commit putting these methods back in the original order?